lib/puma/dsl.rb

```diff
@@ -1204,7 +1202,7 @@ def warn_if_in_single_mode(hook_name)
         log_string =
           "Warning: You specified code to run in a `#{hook_name}` block, " \
           "but Puma is not configured to run in cluster mode (worker count > 0 ), " \
-          "so your `#{hook_name}` block did not run"
+          "so your `#{hook_name}` block will not run."
```

Comment:
This warning is displayed during configuration, so it makes more sense to tell the user that it will not run, as the hooks haven't actually been processed yet.